e5-cert.yaml

```diff
@@ -2,20 +2,20 @@ ceremony-type: intermediate
 pkcs11:
   module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
   pin: 1234
-  signing-key-slot: 1094195990
+  signing-key-slot: 1307844626
   signing-key-label: root-x2
 inputs:
   issuer-certificate-path: root-x2.cert.pem
-  public-key-path: int-e1.key.pem
+  public-key-path: int-e5.key.pem
 outputs:
-  certificate-path: int-e1.cert.pem
+  certificate-path: int-e5.cert.pem
 certificate-profile:
   signature-algorithm: ECDSAWithSHA384
-  common-name: Example ECDSA 1
-  organization: Example
+  common-name: (FAKE) E5
+  organization: (FAKE) Let's Encrypt
   country: XX
```

Comment:
The idea was that we want there to be zero chance of the certs generated by this demo of ever being mistaken for the real thing. Since the countryName field must be a string of length 2, we can't do "(FAKE) US", so we went with "XX" (a reserved code that is guaranteed to never be assigned to a real country) instead.

But yeah it does mean that there's one more place where these configs have to be edited in order to become the real thing.

Reply:
Yeah! I was trying to strike a middle ground -- personally I think "(FAKE) Let's Encrypt" is just as good as "example.com" for ensuring there is no confusion, which is why I changed that one back. I would have changed the country to "(FAKE) US" if I could :D